content/languages/ocaml/recipes.md

```diff
@@ -31,4 +31,4 @@ Completed in 0.24ms
 ### References
 
 - [Improving OUnit Output](https://cs3110.github.io/textbook/chapters/data/ounit.html#improving-ounit-output)
-- [OUnit: xUnit testing framework for OCaml](https://gildor478.github.io/ounit/ounit2/index.html#error-reporting)
\ No newline at end of file
+- [OUnit: xUnit testing framework for OCaml](https://ocaml.org/p/ounit2/2.2.3/doc/index.html#error-reporting)
```

Comment:
It seems that only the  `2.2.6` version of OUnit has docs on ocaml.org, all the other versions return a 404.

Therefore the version number should be changed in the URL: `2.2.3` --> `2.2.6`.
Same problem at the page  `https://docs.codewars.com/languages/ocaml`: it points to `2.2.7`